Doc/library/cmd.rst

```diff
@@ -42,6 +42,9 @@ interface.
    :attr:`use_rawinput` attribute to ``False``, otherwise *stdin* will be
    ignored.
 
+   .. versionchanged:: 3.11
+      ``completekey='tab'`` is replaced by ``'^I'`` for ``editline``.
```

Comment:
Because the other note has three blank spaces, I appended them!
```
$grep -rin versionchanged -C2
./library/exceptions.rst-756-   :py:const:`~errno.EPERM`, and :py:const:`~errno.ENOTCAPABLE`.
./library/exceptions.rst-757-
./library/exceptions.rst:758:   .. versionchanged:: 3.11.1
./library/exceptions.rst-759-      WASI's :py:const:`~errno.ENOTCAPABLE` is now mapped to
./library/exceptions.rst-760-      :exc:`PermissionError`.
```